superset/views.py

```diff
@@ -2263,7 +2273,6 @@ def show_traceback(self):
             title=ascii_art.stacktrace,
             art=ascii_art.error), 500
 
-    @has_access
```

Comment:
I would keep @has_access for welcome too and would add it by default to the public role